### PR TITLE
Add Ink! playground maintenance

### PR DESCRIPTION
**Pull request by radhe-zeeve**
#### Ink! playground

Maintain Ink! playground repository and hosting of Ink! playground portal

#### Grant [level](https://github.com/w3f/Grants-Program#level_slider-levels)

- [ ] **Level 1**:  Up to $10,000, 2 approvals
- [x] **Level 2**:  Up to $30,000, 3 approvals
- [ ] **Level 3**:  Unlimited, 5 approvals (for >$100k: Web3 Foundation Council approval)

#### Application Checklist

- [x] The [application template](https://github.com/w3f/Grants-Program/blob/master/applications/application-template.md) has been copied and aptly renamed (`project_name.md`).
- [x] I have read the [application guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/grant_guidelines_per_category.md).
- [x] Payment details have been provided (Polkadot AssetHub (DOT, USDC & USDT) address in the application and bank details via email, if applicable).
- [x] I am aware that, in order to receive a grant, I (and the entity I represent) have to successfully complete a KYC/KYB check.
- [x] The software delivered for this grant will be released under an open-source license specified in the application.
- [x] The initial PR contains only one commit (squash and force-push if needed).
- [x] The grant will only be announced once the first milestone [has been accepted](https://github.com/w3f/Grant-Milestone-Delivery#process) (see the [announcement guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/Support%20Docs/announcement-guidelines.md)).
- [ ] I prefer the discussion of this application to take place in a private Element/Matrix channel. My username is: `@_______:matrix.org` (change the homeserver if you use a different one)

### CODE REVIEW

**Comment by PieWol** [comment]
Hey @radhe-zeeve,
thank you for your application. As the ink! team is moving towards decentralization, there are ongoing discussions about the future of ink!. These discussions may impact the decision period for this grant, which is expected to be longer than usual. I wanted to inform you of this.

File: applications/ink-playground-maintance.md [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+  - expected 100 GB egress
+- The expected additional cost will be $240/month for Hosting
+
+### Overview
+
+- **Start Date:** 25th March 2024
+- **Sprint/Period Duration:** 4 Weeks
+- **Total Duration:** 1 year
+- **Full-Time Equivalent (FTE):**  2 FTE
+- **Max budget per sprint/period:** 6,000 USD (excluding server changers, if any)
```
Comment (semuelle) [inline applications/ink-playground-maintance.md:96]:
```suggestion
- **Max budget per sprint/period:** 6,000 USD (excluding server charges, if any)
```

**Review by semuelle — CHANGES_REQUESTED** [review]
Thanks for the follow-up application, @radhe-zeeve. You marked this as Level 2, but from what I'm reading this would be 13x6000 USD, so Level 3. Or did I misunderstand this?

File: applications/ink-playground-maintance.md (hunk 1 above)
Comment (gnv-zeeve) [reply applications/ink-playground-maintance.md:96]: @radhe-zeeve Please accept the spelling mistake change. Thanks @semuelle We must have gotten confused, now I realise the amount for level is to be in total.

Do you advise that we make it 6 months or rather change the level?

Comment (semuelle) [reply applications/ink-playground-maintance.md:96]: Fair question, @radhe-zeeve. With the handover of ink!, it's currently a bit of guesswork what will be needed. We could either sign a grant for a shorter period, or we wait until we get some feedback from the (new) ink! team.

**Review by semuelle — COMMENTED** [review]
Hi @radhe-zeeve. Sorry for the long wait. Given that it's currently unclear who will take over ink! development and how, I suggest that we put this application on hold until we can get some input on the playground from the new team.

**Review by Noc2 — CHANGES_REQUESTED** [review]
Thanks for the application and all the effort that you put into this. However, given that the future of ink! is still unclear, so we decided to close the application for now. We will ping you as soon as we have more information.

**Comment by Noc2** [comment]
(see above)

**Pull request closed without merging**